src/ucp/core/ucp_context.c

```diff
@@ -476,6 +477,12 @@ static ucs_config_field_t ucp_context_config_table[] = {
    ucs_offsetof(ucp_context_config_t, reg_nb_mem_types),
    UCS_CONFIG_TYPE_BITMAP(ucs_memory_type_names)},
 
+  {"POPULATE_NONBLOCK_MEMTYPES", "try",
```

Comment:
we already have REG_NONBLOCK_MEM_TYPES for this purpose.
We thought to add platform specific config, similar to what @rakhmets implemented in #9776